.github/workflows/test-sphinx.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      python-version: '3.10'
+    strategy:
+      matrix:
+        tasks: [ 'sphinx' ]
+        sphinx-version: [ '5.3.0', '6.2.1', '7.*' ]
```

Comment:
The `tasks` entry is not really needed, but a workaround to get the word `sphinx` included in the description of the test:

![image](https://github.com/user-attachments/assets/7270784b-bba6-4e3c-ba66-088347286c63)

An alternative would be to write:
```yaml
sphinx: [ 'sphinx==5.3.0', 'sphinx==6.2.1', 'sphinx==7.*' ]
```
and adjust the rest of the file accordingly, but I like the current approach more.

Reply:
I changed my mind about this. Now we have

![image](https://github.com/user-attachments/assets/55eb0ce2-b174-48c0-ab80-3264684bfb67)

as we also do with `pandas` in `audb`:

![image](https://github.com/user-attachments/assets/2d96a101-b05e-4971-b8b7-cd3010303d07)